git-annex-remote-rclone

```diff
@@ -216,21 +216,24 @@ while read -r line; do
 			# Any nonzero object count means present.
 			# Some rclone backends support multiple
 			# files under one name.
-			if check_result=$(rclone size "$LOC$key" 2>&1) &&
-				echo "$check_result" | GREP -E 'Total objects: [1-9][0-9]*\>'; then
-				echo CHECKPRESENT-SUCCESS "$key"
+			if check_result=$(rclone size --json "$LOC$key" 2>&1) &&
```

Comment:
`--json` was added in v1.45~74 if I get it right from `git log` so we need to drop old versions from testing within .github/workflows/test.yml  and likely to note something in README as for that to be the minimal version supported.  

Unlikely we should do here cumbersome version comparison to alert user

Reply:
I dropped the older versions from the tests and removed logic for now removed versions. I added a testing for the affected commands. I also updated the README, please check if it matches your plans surrounding supported version.